packages/x-charts/src/themeAugmentation/themeAugmentation.spec.ts

```diff
@@ -188,5 +188,53 @@ createTheme({
       //   constent: { color: 'red' },
       // },
     },
+    MuiPieArc: {
+      defaultProps: {
+        id: 'toto',
+        // @ts-expect-error invalid MuiMarkElement prop
+        someRandomProp: true,
+      },
+      styleOverrides: {
+        root: { backgroundColor: 'red' },
```

Comment:
In this case not even the `root` slot works, because respective `overridesResolver` uses `styles.arc` instead of `styles.root`. 🙈